pydra/engine/task.py

```diff
@@ -348,10 +346,37 @@ def _command_args_single(self, state_ind, ind=None):
                 cpath = cdir.joinpath(lpath.name)
                 value = str(cpath)
             if f.type is bool:
+                if "argstr" in f.metadata:
+                    cmd_add.append(f.metadata["argstr"])
```

Comment:
shouldn't this happen only if value is True

Reply:
should change, will be cleaner